test/screenshot_integration/force_screenshot.test.js

```diff
@@ -0,0 +1,24 @@
+import { browser } from '../../';
+import assert from 'assertive';
+
+describe('forced screenshot', () => {
+  before(browser.beforeHook());
+
+  it('my test', () => {
+    this.browser.navigateTo('/');
```

Comment:
this test fails here instead of 2 lines below because it shouldn't reference `this`